### PR TITLE
[Data Team Helper] "Fixed" relevant warnings for GET and filtering

### PR DESCRIPTION
**Pull request by AnyhowStep**
LorisForm spams when filtering, this is fixed by #2810 , not this pull request.

This pull request literally just spams `isset($x) ? $x : null` all over to get rid of the warnings. The logic should still be the same. However, with so much accidental access to undefined indices... I don't trust the logic of the code at all.

Added variables with `null` values where warnings said there was access to undefined variables.

-----

Line 58 in this pull request is of note,

Changed `$DB->select($query,  $info);`

To `$info  = $DB->pselect($query, array());`

Unsure if this is the recommended approach. Needs another set of eyes.

### CODE REVIEW

**Comment by jstirling91** [comment]
@AnyhowStep Would it make sense to create a helper function which takes in the variable and key as parameters and does the isset in there returning wither the value or null? Also this has conflicts

**Comment by driusan** [comment]
There's no need for a special function, PHP has a null coalesce operator since PHP7, which is now the minimum LORIS version supported: https://wiki.php.net/rfc/isset_ternary

**Comment by jstirling91** [comment]
@driusan In that case I think we should enforce using that as opposed to the `isset($var) ? $var : null`

**Comment by AnyhowStep** [comment]
Needs @kongtiaowang 's pull request to be merged first or I'll just keep failing Travis. Also needs to be rebased to match the new `$sanitize` variable.

**Comment by AnyhowStep** [comment]
Note to self, #2780 is where `$sanitize` was added. Looking at it might help understand motivation for it and ease rebase process.

**Comment by AnyhowStep** [comment]
Merged conflicts with web editor, will probably fail phpcs. Waiting for Travis.

**Comment by AnyhowStep** [comment]
I just noticed that the pull request to remove 5.6 has been merged. This is no longer blocked.

File: modules/data_team_helper/php/NDB_Form_data_team_helper.class.inc [hunk 1]
```diff
@@ -58,23 +58,29 @@ class NDB_Form_Data_Team_Helper extends NDB_Form
             $qparams =array();
             $info    =$DB->pselect($query,  $qparams);
             foreach ($info as $data) {
-                $percent_completed            = $data['Name']
-                    . ":" . $data['Visit_label']
+                $percent_completed            = (isset($data['Name']) ?
```
Comment (jstirling91) [inline modules/data_team_helper/php/NDB_Form_data_team_helper.class.inc]: Switch these to use null operator like below

**Comment by kongtiaowang** [comment]
@AnyhowStep 
When I tested it got this "PHP Parse error:  syntax error, unexpected '?' in /var/www/loris/modules/data_team_helper/php/NDB_Form_data_team_helper.class.inc on line 61
[Thu Jun 08 15:18:33.212627 2017] [:error] [pid 15121] [client 132.216.56.196:52346] PHP Stack trace:"

**Comment by AnyhowStep** [comment]
PHP 5 does not have the null coalesce operator.

**Comment by johnsaigle** [comment]
@jstirling91 could you approve the changes?

**Comment by AnyhowStep** [comment]
Wait, @kongtiaowang just showed me a problem. I need to check it first.

**Comment by AnyhowStep** [comment]
Shen had to install `libapache2-mod-php7.0` and run `a2dismod php5`, `a2enmod php7.0`. Then, `sudo apt-get install php7.0-xml` (to open the `config.xml` file) and then, `sudo service apache2 restart`.

**Comment by jstirling91** [comment]
Moving to senior review based on @kongtiaowang adding PassManualTests

**Pull request merged**